distribution/lib/Standard/Database/0.0.0-dev/src/Data/SQL_Statement.enso

```diff
@@ -43,8 +43,12 @@ type SQL_Statement
         strings = self.internal_fragments . map <| case _ of
             SQL_Fragment.Code_Part code -> code
             # TODO at some point we may try more sophisticated serialization based on data type
+            # TODO date and time formatting is limited and will lose sub-second precision and timezone offset.
```

Comment:
Cannot we change the format to not lose this data? Or is there some deeper reason for that? If so, can it be explained?
Are there plans to address this issue later? Would be good if some additional information was provided here.